iOS/Flows/Threads/ThreadViewController.swift

```diff
@@ -25,7 +25,12 @@ class ThreadViewController: DiffableCollectionViewController<MessageSequenceSect
     
     let blurView = BlurView()
     let parentMessageView = MessageContentView()
-    
+
+    weak var messageCellDelegate: MesssageCellDelegate? {
+        get { return self.dataSource.messageCellDelegate }
+        set { self.dataSource.messageCellDelegate = newValue }
+    }
```

Comment:
The thread coordinator will eventually provide the message cell delegate to show attachments, edit messages, etc.